docs/guides/blocks/formulas.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Formulas
+---
+
+# Add custom formulas
+
+Formulas are one of the most basic building blocks in Coda; used to calculate values, filter tables, and so much more. Coda provides a wide array of [built-in formulas][formulas], and using Packs you can add your own custom formulas. Once your Pack is installed in a doc you can use those custom formulas anywhere, intermingling them with built-in formulas or those from other Packs.
+
+[View Sample Code][samples]{ .md-button }
+
+
+## Structure of a formula
+
+TODO
+
+
+## Naming
+
+The name of a formula can only contain the letters, numbers, and underscores. By convention these names should by written in upper camel case, like `DoSomethingCool`.
+
+Formula names must be unique within a Pack, but can be the same as built-in formulas or those in other Packs. When a doc has access to multiple formulas with the same name the Pack's icon is used to distinguish them.
+
+<img src="../../../images/formula_disambiguation.png" srcset="../../../images/formula_disambiguation_2x.png 2x" class="screenshot" alt="Icons used to disambiguate formulas">
+
+
+## Parameters
+
+Formulas can accept parameters, which is the primary way for them to access data from the document. See the [Parameters guide][parameters] for more information and examples.
```

Comment:
May want to mention, that even for action formulas, it's a best practice to return a non-empty result.

Reply:
I've just put a TODO here for now, since I've got a lot of work to do with actions and return types and haven't quite figured out how I'll structure it yet.